docs/resources/Channel.md

```diff
@@ -562,15 +562,17 @@ An object that represents a tag that is able to be applied to a thread in a `GUI
 
 ###### Forum Tag Structure
 
```

Comment:
```suggestion
###### Forum Tag Structure

> info
> When updating a `GUILD_FORUM` channel, tag objects in `available_tags` only require the `name` field.

```

ok yah sorry for another change, but I like @advaith1's suggestion of pulling it to the top—i think the footnote would be easy to miss